tasks/omnibus.py

```diff
@@ -517,6 +517,277 @@ def _packages_from_deb_metadata(lines: Iterator[str]) -> Iterator[DebPackageInfo
         yield DebPackageInfo.from_metadata(package_info)
 
 
+@task(
+    help={
+        'arch': "Target architecture: 'arm64' or 'amd64' (default: auto-detect from host)",
+        'cache-dir': "Base directory for caches (default: ~/.omnibus-docker-cache)",
+        'compress': "Compress the package with XZ (slower but smaller). Default: False for local builds",
+        'workers': "Number of parallel workers for compression and builds (default: 8)",
+        'container-name': "Docker container name (default: ddagentbuilder)",
+        'build-image': "Docker build image to use (default: uses version from .gitlab-ci.yml)",
+        'tag': "Tag for the built Docker image (default: datadog-agent:local)",
+    }
+)
+def docker_build(
+    ctx,
+    arch=None,
+    cache_dir=None,
+    compress=False,
```

Comment:
If we stick to this strictly to outputting a docker image (and never a package, for example), it looks like there's nothing to gain from making this configurable).